tutorials/camkes-vm-kernel-module/camkes-vm-kernel-module.md

```diff
@@ -1,7 +1,7 @@
 <!--
   Copyright 2021 Michael Neises
 
-  SPDX-License-Identifier: BSD-2-Clause
+  SPDX-License-Identifier: CC-BY-SA-4.0
```

Comment:
CC-BY-SA-4.0 seems to be compatible with BSD-2-Clause, so the above change is probably legal.

Doesn't seem very polite to do though, especially if there are no other changes.